src/client/src/rt-platforms/browser/utils/sendNotification.ts

```diff
@@ -8,18 +8,31 @@ export interface NotificationMessage {
   tradeNotification: Trade
 }
 
+let registration: ServiceWorkerRegistration
+navigator.serviceWorker &&
+  navigator.serviceWorker.ready.then(reg => {
+    registration = reg
+  })
+
 export const sendNotification = ({ tradeNotification }: NotificationMessage) => {
   const status = tradeNotification.status === 'done' ? 'Accepted' : 'Rejected'
   const title = `Trade ${status}: ${tradeNotification.direction} ${tradeNotification.dealtCurrency} ${tradeNotification.notional}`
   const body = `vs. ${tradeNotification.termsCurrency} \nRate ${tradeNotification.spotRate}    Trade ID ${tradeNotification.tradeId}`
```

Comment:
Shouldn't the title and body text of the notifications change as per the designs linked in the JIRA?
![image](https://user-images.githubusercontent.com/3295115/98541532-80b1d700-2287-11eb-9466-906740c5f9f2.png)
This was the main part of the JIRA